core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemConverter.java

```diff
@@ -73,10 +74,32 @@ public void expand(List<CodeSystem> results) {
 		expandAvailableUpgrades(results);
 		expandExtensionOfBranchState(results);
 		expandUpgradeOfInfo(results);
+		expandVersions(results);
 		
 		// expand additional fields via pluggable converters
 		converters.expand(context(), expand(), locales(), results);
 	}
+	
+	private void expandVersions(List<CodeSystem> results) {
```

Comment:
We already have this feature implemented in the generic resource API. Would it be possible to reuse that logic here as well?
See `com.b2international.snowowl.core.request.ResourceConverter.expandVersions(List<Resource>)`